scripts/ol_init.sh

```diff
@@ -0,0 +1,3 @@
+curl -XPUT http://localhost:9200/_cluster/settings -H 'Content-type: application/json' --data-binary $'{"transient":{"cluster.max_shards_per_node":10000}}'
```

Comment:
It might be useful to add an ol_wipe script that deletes all your indices + drops/recreates your database? I can imagine people calling init expecting that it would reset your setup if there's no separate wipe script